ftp/OlsonJJ/recuerdos-luna/recuerdos-luna.ly

```diff
@@ -0,0 +1,261 @@
+\version "2.18.2"
+
+\header {
+ title = "Recuerdos de la Luna"
+ subtitle = \markup { \center-column { { "Un arreglo sencillo de “Recuerdos de la Alhambra” por Francisco Tárrega" } { "en homenaje a “Claro de Luna” por Beethoven" } } }
+ %composer = "Arreglo:  Jeffrey Olson"
+ %arranger = "Dedos:  Dennis Burns"
+ composer = ##f
+ arranger = \markup { \abs-fontsize #10 \right-column { { "Arreglo:  Jeffrey Olson" } { "Dedos:  Dennis Burns" } } }
+ myfoot = ##f
+ style = "Romantic"
+ date = "1802,1896,2017" % (by Beethoven, Tárrega, Olson)
+ mutopiadate = "1896"
+ source = "Madrid: Orfeo Tracio, n.d. (ca.1920). Plate 1102" % via IMSLP24052
+ mutopiasource = "Madrid: Orfeo Tracio, n.d. (ca.1920). Plate 1102"
+ moreInfo = "This simple arrangement combines Tárrega's “Recuerdos de la Alhambra” 
+            with the triplets of Beethoven's “Moonlight Sonata”.  It was originally developed
+            as one of many sight reading exercises in “Chromatic Fretboard Exercises” to be
+            playable in a single fixed position (either VII or IX), but it sounds best
+            with the indicated fingering by Dennis Burns which allows each triplet arpeggio
+            to ring freely." 
+
+ %            o_
+ %       (\___\/_____/)
+ %  ~ ~ ~ ~ ~ / ~ ~ ~ ~ ~ ~ ~ 
+ maintainer = "Jeffrey Olson & Dennis Burns"
+ maintainerEmail = "gmail's jjocanoe & boldersounds.com's dennis"
+ license = "Public Domain" 
+
+ mutopiacomposer = "OlsonJJ"
+ mutopiatitle = "Recuerdos de la Luna"
+ mutopiainstrument = "Guitar"
+
+ footer = "Mutopia-2017/10/14-draft"
+ copyright =  \markup { \override #'(baseline-skip . 0 ) \right-column { \sans \bold \with-url #"http://www.MutopiaProject.org" { \abs-fontsize #9  "Mutopia " \concat { \abs-fontsize #12 \with-color #white \char ##x01C0 \abs-fontsize #9 "Project " } } } \override #'(baseline-skip . 0 ) \center-column { \abs-fontsize #11.9 \with-color #grey \bold { \char ##x01C0 \char ##x01C0 } } \override #'(baseline-skip . 0 ) \column { \abs-fontsize #8 \sans \concat { " Typeset using " \with-url #"http://www.lilypond.org" "LilyPond" " by " \maintainer " " \char ##x2014 " " \footer } \concat { \concat { \abs-fontsize #8 \sans{ " Placed in the " \with-url #"http://creativecommons.org/licenses/publicdomain" "public domain" " by the typesetter " \char ##x2014 " free to distribute, modify, and perform" } } \abs-fontsize #13 \with-color #white \char ##x01C0 } } }
+ tagline = ##f
```

Comment:
The header tags, **footer** and **copyright**, are created on publication and overwritten every time this piece is revised. It is probably best to leave them out of a new submission and not edit them when revised.